### ISSUE TITLE
wrong spectra of binaries

### ISSUE THREAD

**Issue by elisadelgado** [njcuk9999/apero-utils#233]
Hello
In the NIRPS-OS1 program there are some mixed spectra for a couple of M dwarfs (for which the primary was observed instead). Would it be possible to change the name of the header (or any other trick) so these spectra go to the primary object list? Like that we will have a better spectrum for the primary and most importantly the correct combined template for the M dwarf. These are the 2 cases:

GJ199B:
NIRPS_2024-03-07T02_14_29_287--> this spectrum belongs to GJ199A

GJ872B:
NIRPS_2023-06-28T09_29_30_865 --> remove this spectrum from database, seems wrong

move the following 4 spectra to GJ872A
NIRPS_2023-08-27T06_02_21_947
NIRPS_2023-09-01T04_59_46_345
NIRPS_2023-09-01T05_14_54_781
NIRPS_2023-11-03T02_48_51_801

thanks

**Comment by elisadelgado** [comment]
Thanks for the fast reply Neil. Ok, I will try with ESO.
About GJ527B, don't worry. In that case the observer chose Tau Boo, but it's because it's almost impossible to see the M dwarf. Those spectra can be even removed because they are saturated.
In the case of GJ199B and GJ872B the (wrongly taken) spectra of the primaries are not saturated (at least for GJ199A can be used). I have doubts about GJ872B, because the telluric correction is not good in the spectra with the highest S/N (from August and September).

**Comment by elisadelgado** [comment]
The files' header were corrected in the ESO archive, they were fast! Let me know when is the next sync so I check how the spectra and templates look. thanks

**Comment by njcuk9999** [comment]
Wow that was fast!

I'll tag @vandalt here. Thomas do we need a full list of files to re-sync these or can your script look for changes in meta data?

**Comment by njcuk9999** [comment]
Could you (@elisadelgado) also request Tau Boo - even if the data is bad it will never be removed from ESO so it might as well be flagged as the correct target!

**Comment by elisadelgado** [comment]
ok, I will also ask to change the names for those.
Then, I should also ask to change to GJ872A the spectra NIRPS_2023-06-28T09_29_30_865, since given the flux it should be from the primary as well. I think this one is saturated or I don't know. But cannot be used for GJ872A.

By the way Neil, could it be possible to get a finding chart for GJ199B as the ones for the other stars in APERO? I got an image from the HARPS pointing last week, but it doesn't look like the finding chart from ESO-p2. So I still have doubts if the star is the correct one. For GJ426B=HD98736B (a similarly close binary), the pointing image was very similar to the APERO finding chart, so it was very useful for the observer, but since GJ199A was added later to ARI database, the finder chart was not created I believe.

**Comment by njcuk9999** [comment]
Re: the finder charts, they should be updated. Let me know if any are missing!

**Comment by elisadelgado** [comment]
ahh, I see it now, but it's from another star!! The finding chart coordinates are  
RA: 2:17:18.95550353 Dec: 33:50:48.64100136
but this star is at RA:05 18 47.5387663944 DEC: -21 23 36.484311516
that's strange (or I have a special touch to find all weird stuff, hahaha)

**Comment by elisadelgado** [comment]
Indeed, also the finding charts from GJ157A, GJ157B and GJ166 have wrong coordinates. All these stars were recently added in the list since I started observing them by February

**Comment by njcuk9999** [comment]
Strange! Could you check them in the astrometric database? Are they correct in there?

**Comment by elisadelgado** [comment]
yes, the coordinates are correct both in the ha-online table and in the google sheet with all the targets. I also noticed GJ9063A has wrong finding chart, but again good coordinates

**Comment by njcuk9999** [comment]
Okay I see the problem, it seems the name of the object is sorted alphabetically and the table or ra/dec etc is not... seems like previous both were sorted. I'll fix this but just to be safe I'll re-run all finder charts (which will take half a day). So tomorrow this _should_ be fixed.

**Comment by njcuk9999** [comment]
Okay I looked at GJ199B  and it looks correct. So at least the finding charts should be good now!

**Comment by elisadelgado** [comment]
thanks a lot, yes, they are fine now :)
